test/acceptance/kafo_configure_test.rb

```diff
@@ -52,6 +52,15 @@ module Kafo
       end
     end
 
+    describe '--list-scenarios' do
+      it 'lists scenarios' do
+        code, out, err = run_command '../bin/kafo-configure --list-scenarios'
+        _(code).must_equal 0, err
+        _(out).must_include "Available scenarios"
+        _(out).must_include "default \e[0m(use: --scenario default)\n"
```

Comment:
seems the tests don't use colors and fail on the difference.

Reply:
Darn dynamic calculation based on environment. I set it to `--no-colors` to be explicit.